.github/workflows/agent-pr.yml

```
@@ -22,16 +22,25 @@ jobs:
           git config --global user.name Netdatabot
           git config --global user.email bot@netdata.cloud
       - name: Create Branch
-        run: git checkout -b agent-${{ github.event.inputs.agent_version }}
+        run: |
+          git checkout -b agent-${{ github.event.inputs.agent_version }}
+          git push -u origin agent-${{ github.event.inputs.agent_version }}
       - name: Update Files
         run: .github/scripts/update-agent-version.py ${{ github.event.inputs.agent_version }}
       - name: Commit Changes
-        run: git commit -a -m 'Update agent version to ${{ github.event.inputs.agent_version }}.'
-      - name: Push Branch
-        run: git push -u origin agent-${{ github.event.inputs.agent_version }}
+        uses: swinton/commit@v2.x
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          files: |
+            charts/netdata/Chart.yaml
+            charts/netdata/README.md
+          commit-message: 'Update agent version to ${{ github.event.inputs.agent_version }}.'
+          ref: refs/heads/agent-${{ github.event.inputs.agent_version }}
       - name: Create PR
         uses: repo-sync/pull-request@v2
         with:
           source_branch: agent-${{ github.event.inputs.agent_version }}
           pr_title: 'Update agent version to ${{ github.event.inputs.agent_version }}.'
```

```suggestion
          pr_title: 'Update agent version to ${{ github.event.inputs.agent_version }}'
```

neat: `Update agent version to v1.30.1.` => `Update agent version to v1.30.1` (https://github.com/netdata/helmchart/pull/203)

I think it looks better w/o `.`.